bucket/windiff.json

```diff
@@ -0,0 +1,13 @@
+{
+    "version": "2.1.0",
+    "description": "The omnipotent, omniscient, omnipresent diffing program",
+    "homepage": "https://en.wikipedia.org/wiki/WinDiff",
+    "license": "UNLICENSED",
+    "url": "https://github.com/kitamstudios/WinDiff/releases/download/v2.1.0/WinDiff.zip",
+    "hash": "e25ccdc2fef77eb001fc3a5d755d448a301338d7e4feb2cf365bcb3d34200931",
+    "bin": "bin\\WinDiff.exe",
+    "checkver": {
+        "github": "https://github.com/kitamstudios/WinDiff",
+        "regex": "/releases/tag/v([\\d.]+)"
+    }
```

Comment:
```suggestion
    },
    "autoupdate": {
        "url": "https://github.com/kitamstudios/WinDiff/releases/download/v$version/WinDiff.zip"
    }
```

Reply:
verified that both checkver and autoupdate urls resolve correctly with the change.